csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase[T].cs

```diff
@@ -6,36 +6,32 @@
 namespace Platform.Collections.Segments.Walkers
 {
     /// <summary>
-    /// <para>
-    /// Represents the all segments walker base.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents the base abstract class for walkers on all elements.</para>
+    /// <para>Представляет базовый абстрактный класс для проходчиков по всем элементам.</para>
     /// </summary>
     /// <seealso cref="AllSegmentsWalkerBase{T, Segment{T}}"/>
     public abstract class AllSegmentsWalkerBase<T> : AllSegmentsWalkerBase<T, Segment<T>>
     {
         /// <summary>
-        /// <para>
-        /// Creates the segment using the specified elements.
-        /// </para>
-        /// <para></para>
+        /// <para>Initializes a new instance of the <see cref="Segment"/> class.</para>
+        /// <para>Инициализирует новый экземпляр класса <see cref="Segment"/> .</para>
         /// </summary>
         /// <param name="elements">
-        /// <para>The elements.</para>
-        /// <para></para>
+        /// <para>List of elements.</para>
+        /// <para>Список  элементов.</para>
         /// </param>
         /// <param name="offset">
-        /// <para>The offset.</para>
-        /// <para></para>
+        /// <para>An offset relative to the <paramref name="elements"/> list from which the segment starts.</para>
+        /// <para>Смещение относительно списка <paramref name="elements"/>, с которого начинается сегмент.</para>
         /// </param>
         /// <param name="length">
-        /// <para>The length.</para>
-        /// <para></para>
+        /// <para>A segment's length.</para>
+        /// <para>Длина сегмента.</para>
         /// </param>
         /// <returns>
-        /// <para>A segment of t</para>
-        /// <para></para>
-        /// </returns>
+        /// <para>An instance of the <see cref="Segment{T}"/> class.</para>
+        /// <para>Экземпляр класса <see cref="Segment{T}"/> .</para>
+        ///</returns>
```

Comment:
```suggestion
        /// </returns>
```

Reply:
You also have extra spaces on lines 32 and 33